### ISSUE TITLE
plotting fails

### ISSUE THREAD

**Issue by inijman** [rgcgithub/clamms#10]
HI,

I'm trying to plot events found in a sample but the script fails with the following error:
***** ERROR: Requested column 4, but database file tmp.$SAMPLE.prev-and-next.windows.bed only has fields 1 - 0.

I've found that this file is actually empty, mainly because of the the awk command in plot_cnv.sh is selecting $4==0 from the models.bed file. In my models files only the values, 3,6 and -1 appear.

How can I fix this?

Best,

**Comment by rgcgithub** [comment]
Good catch, we haven't updated the plotting script in quite some time. In the short term, you should be able to change it to '$4 >= 0' (-1 indicates a blacklisted window, which is what this is attempting to filter on). I'll push a fix, but the script may need some more attention.

**Comment by lakhujanivijay** [comment]
Hi 

I am looking at [this](https://github.com/rgcgithub/clamms#visualizing-cnvs-for-a-sample) for plotting the CNV results and I am wondering what is this file - `sample.cnv.txt` ; is this the sample.cnv.bed file which is the output from `$CLAMMS_DIR/call_cnv` program?

Please help.

**Comment by katherinef** [comment]
Yes, it takes the sample.cnv.bed. 

I think I've noticed another problem with the plot_cnvs.R.  It seems to me that the line:
df$max.cn.considered <- ifelse(df$max.cn.considered == 0, 3, -1)
should be:
df$max.cn.considered <- ifelse(df$max.cn.considered >= 0, 3, -1)

Can someone please confirm?

Best wishes

Kath
